simpleclient/src/main/java/io/prometheus/client/Collector.java

```diff
@@ -77,12 +77,18 @@ public String toString() {
       public final List<String> labelNames;
       public final List<String> labelValues;  // Must have same length as labelNames.
       public final double value;
+      public final Long timestamp;
```

Comment:
timestampMs would be clearer.

Can you also add a note that this field is subject to change?

Reply:
I'm agree, this has been changed.